dll/win32/shell32/droptargets/CFSDropTarget.cpp

```diff
@@ -547,104 +547,124 @@ HRESULT CFSDropTarget::_DoDrop(IDataObject *pDataObject,
 
         if (bLinking)
         {
-            WCHAR wszTargetPath[MAX_PATH];
-            WCHAR wszPath[MAX_PATH];
-            WCHAR wszTarget[MAX_PATH];
+            WCHAR wszSrcPath[MAX_PATH], wszSaveTo[MAX_PATH];
```

Comment:
Why not keeping the original name `wszTarget` for this `wszSaveTo` buffer? This could allow reduce the diff.

Reply:
I thought the word `target` was vague for indication.

Reply:
Fixed.